doc/guide/cert-authentication.xml

```diff
@@ -63,6 +53,81 @@ pkcs15-init --store-private-key alice.p12 --format pkcs12 --auth-id 01
 
   </section>
 
+  <section id="certauth-server-ipa">
+    <title>Certificate mapping with FreeIPA</title>
+
+    <para>The domain's users get associated to certificates with
+      the <command>ipa user-add-cert</command> command. This expects PEM format, but without the
+      <code>-----BEGIN</code>/<code>-----END</code> markers.  See the
+      <ulink url="https://www.freeipa.org/page/V4/User_Certificates#Feature_Management">
+      FreeIPA User Certificates documentation</ulink>:</para>
+
+<programlisting>
+ipa user-add-cert alice --certificate="$(grep -v ^---- alice.pem)"
+</programlisting>
+
+  </section>
+
+  <section id="certauth-server-ms-ad">
+    <title>Certificate mapping with Microsoft Active Directory</title>
+
+    <para>The domain user certificates get imported into the <code>userCertificate;binary</code>
+      LDAP attribute. This converts the PEM certificate into binary DER form, creates an
```

Comment:
The "This" here reads as it does above, which makes me understand that the act of importing into the attribute magically converts the PEM certificate into binary DER form.  I don't realise until I get to the `:` at the end that the "This" is actually referring to the following example code.

Maybe write "The following example code converts..." instead.

Reply:
Good point, thanks! Fixed.